lib/aggregations/class-aggregation.php

```diff
@@ -126,25 +126,25 @@ protected function extract_query_values( string $key = '' ): array {
 		$query_var = $key ?: $this->get_query_var();
 
 		/**
-		 * Filters extracted query values for a given query var.
+		 * Filters extracted query values for a given aggregation.
 		 *
-		 * @param array  $query_values The array of extracted query values.
-		 * @param string $query_var    The query var being extracted.
+		 * @param array       $query_values The array of extracted query values.
+		 * @param Aggregation $aggregation  The aggregation being processed.
 		 */
 		return apply_filters(
-			'elasticsearch_extensions_query_values',
+			'elasticsearch_extensions_aggregation_query_values',
```

Comment:
thank you for already updating the wiki!

Reply:
👍